common/src/main/nl/jft/common/util/Numbers.java

```diff
@@ -0,0 +1,37 @@
+package nl.jft.common.util;
+
+/**
+ * {@code Numbers} is an utility class for primitive number types.
+ *
+ * @author Oscar de Leeuw
+ */
+public class Numbers {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1118)